zerver/tests/tests.py

```diff
@@ -2129,6 +2130,15 @@ def test_generate_204(self):
         result = self.client_get("/api/v1/generate_204")
         self.assertEqual(result.status_code, 204)
 
+class AuthorsPageTest(ZulipTestCase):
+    def test_endpoint(self):
+        # type: () -> None
+        result = self.client_get("/authors/")
```

Comment:
Let's do one more thing here--let's assert one of the names of the contributors from the fixture shows up here.